src/server/repository/assessmentCycle/data/getTableData.ts

```diff
@@ -69,8 +70,13 @@ export const getTableData = (props: Props, client: BaseProtocol = DB): Promise<R
                   : ''
               }
               and e.col_name is not null
-              -- exclude fao estimated values
-              and e.value ->> 'faoEstimate' is null
+              ${
+                faoEstimates
+                  ? ''
+                  : `
+               -- exclude fao estimated values
+              and e.value ->> 'faoEstimate' is null`
+              }
```

Comment:
Not sure if this is needed?
This is only used for caching the data, right @minotogna ?

We never access the "database data" directly, we always access through redis?